roles/openshift_health_checker/test/action_plugin_test.py

```diff
@@ -94,6 +94,7 @@ def skipped(result):
     {},
 ])
 def test_action_plugin_missing_openshift_facts(plugin, task_vars, monkeypatch):
+    monkeypatch.setattr(plugin, 'load_known_checks', lambda *_: {})
```

Comment:
What is the point of this line?

Reply:
You want the long story? Short story is that this unit test breaks due to actually loading all the checks, when it should just mock loading them.